cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,58 @@
 /// <reference types='cypress' />
+import CheckoutFormPageObject
+  from '../support/pages/checkoutForm.pageObject';
+import HomeAndCataloguePageObject
```

Comment:
Please add homeCatalogue.pageObject to the PR

Reply:
I did not change it, so it is not added to the pool request